work-item-link-type-blackbox_test.go

```diff
@@ -142,11 +153,19 @@ func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemLinkType() {
 	createPayload := s.createDemoLinkType("test-bug-blocker")
 	_, workItemLinkType := test.CreateWorkItemLinkTypeCreated(s.T(), nil, nil, s.linkTypeCtrl, createPayload)
 	require.NotNil(s.T(), workItemLinkType)
+
 	// Check that the link category is included in the response in the "included" array
 	require.Len(s.T(), workItemLinkType.Included, 1, "The work item link type should include it's work item link category.")
 	categoryData, ok := workItemLinkType.Included[0].(*app.WorkItemLinkCategoryData)
 	require.True(s.T(), ok)
 	require.Equal(s.T(), "test-user", *categoryData.Attributes.Name, "The work item link type's category should have the name 'test-user'.")
+
+	// Check that the link category is included in the response in the "included" array
+	require.Len(s.T(), workItemLinkType.Included, 2, "The work item link type should include it's space.")
```

Comment:
@hectorj2f we need this check because we want to make sure that the category is included in the in `included` array of the JSONAPI response. Here's how the array is filled for a single WILT reponse: https://github.com/almighty/almighty-core/blob/9cffea14e89e85749309935944cd5ea72df3bd2a/work-item-link-type.go#L30